dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/FieldSymbolReference.java

```diff
@@ -23,10 +23,18 @@
  */
 @AutoValue
 abstract class FieldSymbolReference implements SymbolReference {
+  
   /**
-   * Returns the field name of the reference.
+   * Returns the name of the referenced field.
    */
   abstract String getFieldName();
+  
+  @Override
+  public String getErrorString() {
+    return this.getTargetClassName() + "." + getFieldName()
```

Comment:
Consider `String.format()` for this one (and the others, frankly). It'd be easier to read.

Reply:
Actually, it looks like these all might follow the same format: "<reference> is not found, referenced from <source>". Suggests a base class implementation.